tcms/testruns/static/testruns/js/get.js

```diff
@@ -1,11 +1,11 @@
-const allExecutionStatuses = {},
-      allExecutions = {},
-      expandedExecutionIds = [],
-      permissions = {
+const allExecutionStatuses = new Map(),
+    allExecutions = {},
+    expandedExecutionIds = [],
+    permissions = {
```

Comment:
sorry for the formatting changes. I can open another PR for them, but this is how the variables and parentheses should be aligned

Reply:
I did it here - https://github.com/kiwitcms/Kiwi/pull/2094
Let's merge #2094 first, I don't wanna bother with reverting this changes one by one